### PR TITLE
Disable versioning logic to prevent orange warnings in deployments

### PR DESCRIPTION
**Pull request by ronichoudhury**

### CODE REVIEW

**Comment by jeffbaumes** [comment]
This does not seem to remove the orange circles in all cases. May be safer to keep the server logic but disable the display logic in the JS code.

File: app/resonantlab/views/overlays/ProjectLibrary/index.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ let ProjectLibrary = ProjectSettings.extend({
           if (d.attributes.meta.rlab.versionNumber === appVersion) {
             return 'none';
           } else {
-            return 'block';
+            return 'none';
```
Comment (jeffbaumes) [inline app/resonantlab/views/overlays/ProjectLibrary/index.js]: DataSetLibrary/index.js has a similar need for a change.

**Comment by ronichoudhury** [comment]
Should I revert the change to the server logic, now that displaying the warning itself is suppressed?

**Comment by jeffbaumes** [comment]
Yeah, I'd back out the server logic change.

**Comment by ronichoudhury** [comment]
@jeffbaumes PTAL

**Pull request merged**